src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -137,14 +137,15 @@ class PanelMediaBrowser extends LitElement {
           --mdc-theme-primary: var(--app-header-text-color);
         }
         ha-media-player-browse {
-          height: calc(100vh - 64px);
+          height: calc(100vh - var(--header-height));
```

Comment:
If we're making these changes here, we should also make sure that we do this for all the layouts in `src/layouts`

Reply:
Need to either adjust the header's, make another var, etc. for other layouts as they require the header's height + 1.

65px rather than 64px. Will check it out more later tonight or tomorrow.